### PR TITLE
Added firestore configuration

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/panel/navbar/navbar.component.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
   justify-content: center;
   min-width: 50px;
   align-items: center;
-  margin: 0 .6rem 0 .6rem
+  cursor: pointer
```
Comment (FilipTalaga) [inline src/app/home/panel/navbar/navbar.component.scss:30]: Semicolon

File: src/app/home/panel/navbar/navbar.component.scss [hunk 2]
```diff
@@ -46,3 +46,28 @@
...
+  .element-container {
+    margin: 0 1rem 0 1rem;
+  }
+}
+
+@media (min-width: 391px) {
+  .element-container {
+    margin: 0 1.2rem 0 1.2rem;
+  }
+}
```
Comment (FilipTalaga) [inline src/app/home/panel/navbar/navbar.component.scss:72]: Do you really need that many different media queries? Those are so close. I guess two shoud be fine.

File: src/app/home/panel/navbar/navbar.component.ts [hunk 3]
```diff
@@ -7,9 +8,25 @@ import { Component, OnInit } from '@angular/core';
...
+  goToWallet() {
+    this.router.navigate(['/wallet']);
+  }
+
+  goToTransactions() {
+    this.router.navigate(['/transactions']);
+  }
+
+  goToBudgets() {
+    this.router.navigate(['/budgets']);
```
Comment (FilipTalaga) [inline src/app/home/panel/navbar/navbar.component.ts:30]: There is some repeatability here, maybe you should try to create a generic method:
```
goTo(path: string) {
    this.router.navigate([path]);
}
```
And then use it on HTML like this: `(click)="goTo('/budgets')"`

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**